pkg/clustermesh/clustermesh.go

```diff
@@ -91,6 +92,8 @@ type ClusterMesh struct {
 	controllers   *controller.Manager
 	configWatcher *configDirectoryWatcher
 
+	IPCache *ipcache.IPCache
```

Comment:
Does this need to be exported?

Reply:
Nope, but I'll add a public one into the Configuration object so that this can be configured at init time. Then I'll hide this one.